File: C#/Eliminacion0/Eliminacion.cs
using System;
using System.Collections.Generic;

// Programa para eliminar un elemento al principio de un array
class Eliminacion
{
    static void Main(string[] args)
    {
        List<int> inputArr = new List<int> { 11, 21, 31, 41, 51, 61 };
        Console.WriteLine("Antes de la eliminacion, el array es: ");
        for (int j = 0; j < inputArr.Count; j++)
        {
            Console.Write(inputArr[j] + " ");
        }

        // Eliminando el primer elemento del inputArr[0]
        Console.WriteLine("\nDespues de la eliminacion, el array es: ");
        inputArr.RemoveAt(0);
        for (int j = 0; j < inputArr.Count; j++)
        {
            Console.Write(inputArr[j] + " ");
        }

        // Eliminando el ultimo elemento del inputArr[-1]
        inputArr = new List<int> { 11, 21, 31, 41, 51, 61 };
        Console.WriteLine("\nDespues de la eliminacion, el array es: ");
        inputArr.RemoveAt(inputArr.Count - 1);
        for (int j = 0; j < inputArr.Count; j++)
        {
            Console.Write(inputArr[j] + " ");
        }

        // Eliminando el numero del medio el inputArr[2]
        inputArr = new List<int> { 11, 21, 31, 41, 51, 61 };
        Console.WriteLine("\nDespues de la eliminacion, el array es: ");
        inputArr.RemoveAt(2);
        for (int j = 0; j < inputArr.Count; j++)
        {
            Console.Write(inputArr[j] + " ");
        }
    }
}

File: Java/eliminacion/Eliminacion.java
import java.util.ArrayList;
import java.util.List;

// Programa para eliminar un elemento al principio de un array
public class Eliminacion {
    public static void main(String[] args) {
        List<Integer> inputArr = new ArrayList<>(List.of(11, 21, 31, 41, 51, 61));
        System.out.println("Antes de la eliminacion, el array es: ");
        for (int j = 0; j < inputArr.size(); j++) {
            System.out.print(inputArr.get(j) + " ");
        }

        // Eliminando el primer elemento del inputArr[0]
        System.out.println("\nDespues de la eliminacion, el array es: ");
        inputArr.remove(0);
        for (int j = 0; j < inputArr.size(); j++) {
            System.out.print(inputArr.get(j) + " ");
        }

        // Eliminando el ultimo elemento del inputArr[-1]
        inputArr = new ArrayList<>(List.of(11, 21, 31, 41, 51, 61));
        System.out.println("\nDespues de la eliminacion, el array es: ");
        inputArr.remove(inputArr.size() - 1);
        for (int j = 0; j < inputArr.size(); j++) {
            System.out.print(inputArr.get(j) + " ");
        }

        // Eliminando el numero del medio el inputArr[2]
        inputArr = new ArrayList<>(List.of(11, 21, 31, 41, 51, 61));
        System.out.println("\nDespues de la eliminacion, el array es: ");
        inputArr.remove(2);
        for (int j = 0; j < inputArr.size(); j++) {
            System.out.print(inputArr.get(j) + " ");
        }
    }
}
